feeds/ipq807x_v5.4/ipq50xx/base-files/etc/init.d/HW-watchdog

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh /etc/rc.common
```

Comment:
please use the kernel based gpio-wdt
https://github.com/Telecominfraproject/wlan-ap/blob/main/feeds/ipq807x_v5.4/ipq60xx/files/arch/arm64/boot/dts/qcom/qcom-ipq6018-cig-wf188.dts#L337